sass/components/_content_project.scss

```diff
@@ -0,0 +1,18 @@
+.ecl-description-list__definition {
```

Comment:
I called it _content_project because it is related to oe_theme_**content_project** module.